app/utils/sentry.ts

```diff
@@ -33,7 +34,20 @@ export function initializeSentry() {
         return;
     }
 
-    Sentry.init({dsn, ...Config.SentryOptions});
+    Sentry.init({
```

Comment:
this is fine, but I do not see any changes in the user & extra context being made, is that going to be addressed?

Reply:
added in app/screens/home/channel_list/channel_list.tsx